SourceX/controls/controller.h

```diff
@@ -19,6 +21,19 @@ class Controller {
 	float leftStickXUnscaled, leftStickYUnscaled, rightStickXUnscaled, rightStickYUnscaled;
 	// Whether stick positions have been updated and need rescaling.
 	bool leftStickNeedsScaling, rightStickNeedsScaling;
+
+	// Returns direction of the left thumb stick or DPad (if allow_dpad = true).
+	AxisDirection GetLeftStickOrDpadDirection(bool allow_dpad = true);
+
+	// Normalize joystick values
+	void ScaleJoysticks();

```

Comment:
Should we consider renaming the method to `NormalizeJoysticks`? The comment seems to indicate the current name is less than ideal.